specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-04-02-preview/privateLinkScopes.json

```diff
@@ -0,0 +1,701 @@
+          },
+          {
+            "$ref": "#/parameters/PrivateLinkScopeName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved private endpoint connections.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/privatelinks.json#/definitions/PrivateEndpointConnectionListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets list of private endpoint connections on a private link scope.": {
+            "$ref": "./examples/PrivateEndpointConnectionList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "TagsResource": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "A container holding only the Tags for a resource, allowing the user to update the tags on a PrivateLinkScope instance."
+    },
+    "KubernetesConfigurationPrivateLinkScope": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Properties that define a Azure Arc PrivateLinkScope resource.",
+          "$ref": "#/definitions/KubernetesConfigurationPrivateLinkScopeProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "An Azure Arc PrivateLinkScope definition."
+    },
+    "KubernetesConfigurationPrivateLinkScopeProperties": {
+      "type": "object",
+      "description": "Properties that define a Azure Arc PrivateLinkScope resource.",
+      "required": [
+        "clusterResourceId"
+      ],
+      "properties": {
+        "publicNetworkAccess": {
+          "description": "Indicates whether machines associated with the private link scope can also use public Azure Arc service endpoints.",
+          "$ref": "#/definitions/PublicNetworkAccessType"
+        },
+        "provisioningState": {
+          "description": "Current state of this PrivateLinkScope: whether or not is has been provisioned within the resource group it is defined. Users cannot change this value but are able to read from it. Values will include Provisioning ,Succeeded, Canceled and Failed.",
+          "type": "string",
+          "readOnly": true,
+          "$ref": "../../common/2022-03-01/definitions.json#/definitions/ProvisioningStateDefinition"
```

Comment:
>2022-03-01 [](http://example.com/codeflow?start=32&length=10)

Recommend adding definitions.json to new API version folder even if it hasn't changed.

Reply:
Can you please explain more how does this help.